control/setup_control.go

```diff
@@ -46,10 +46,12 @@ type PIDLoop struct {
 
 // PIDConfig is values needed to configure a PID control loop.
 type PIDConfig struct {
-	Type string  `json:"type,omitempty"`
-	P    float64 `json:"p"`
-	I    float64 `json:"i"`
-	D    float64 `json:"d"`
+	Type  string  `json:"type,omitempty"`
+	P     float64 `json:"p"`
+	I     float64 `json:"i"`
+	D     float64 `json:"d"`
+	int   float64
+	error float64
```

Comment:
add a comment saying this is the integral sum and error from the pid signal, to be used by the pid controller